File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.cs
public class Solution {
  public int RemoveDuplicates(int[] nums) {
    if (nums.Length < 2) return nums.Length;
    int count = 0;
    for (int i = 1; i < nums.Length; i++) {
      if (nums[i] != nums[count]) {
        count++;
        nums[count] = nums[i];
      } 
    }
    return count+1;
  }
}

File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.py
class Solution:
  def removeDuplicates(self, nums: List[int]) -> int:
    l = len(nums)
    if l == 1: return 1
    slow = 0
    for i in range(1, l):
      if nums[i] != nums[slow]:
        slow += 1
        nums[slow] = nums[i]
    return slow + 1